docs/content/entity-checkout-process.md

```diff
@@ -59,29 +59,30 @@ To perform more complex setup logic, you can add components that themselves act
 
 To use this feature:
-1. Add the `GameObjectInitializationSystem` to your Worker's world.
-2. Add an `Prefab` schema component to SpatialOS entities that should make use of this feature. Set the `prefab` field of the component before the SpatialOS entity is checked-out by your Unity worker (e.g. `prefab = "MyPrefab"`).
-3. Add a `Transform` schema component to SpatialOS entities that should make use of this feature. This is to help the feature determine where the corresponding GameObject shall be spawned in the scene.
-4. Create a prefab for representing your entity in the `Resources` folder of your Unity project. Give it a name, e.g. `MyPrefabClient`, `MyPrefabServer`.
-5. Specify a prefab mapping for matching entities with their corresponding prefabs in `workers\unity\Assets\Playground\Config\PrefabConfig.cs`:
+1. Pass an instance of `EntityGameObjectCreator` with the name of your worker the `GameObjectRepresentationSystemHelper`. In the Playground project setup, this currently happens in the `WorkerUtils` class:
 ```csharp
-public static class PrefabConfig
+public static class WorkerUtils
+{
+    public const string UnityClientType = "UnityClient";
+    public static void AddClientSystems(World world)
     {
-        public static readonly Dictionary<string, PrefabMapping> PrefabMappings = new Dictionary<string, PrefabMapping>
-        {
-            {
-                "MyPrefab",
-                new PrefabMapping { UnityGameLogic = "MyPrefabServer", UnityClient = "MyPrefabClient" }
-            }
-        };
+        // Configuration including the following
+        GameObjectRepresentationSystemHelper.AddSystems(world, new EntityGameObjectCreator(UnityClient));
     }
+}
 ```
+2. Add the `Metadata` schema component to SpatialOS entities that should make use of this feature. Set the `entityType` field of the component before the SpatialOS entity is checked-out by your Unity worker (e.g. `MyEntity`).
+3. Add a `Transform` schema component to SpatialOS entities that should make use of this feature. This is to help the feature determine where the corresponding GameObject shall be spawned in the scene.
+4. Create a prefab for representing your entity in the `Resources\Prefabs` folder of your Unity project. Name it just like in the Metadata component. If you want to use the same prefab for all workers, put it in the `\Common\` subfolder, e.g. `Resources\Prefabs\Common\MyEntity`. If you want to specify different prefabs depending on the workers, put them in subfolders corresponding to the worker type string you gave to the `EntityGameObjectCreator` constructor. In the Playground projects, the worker types are called `UnityClient` and `UnityGameLogic`, so the prefabs would be `Resources\Prefabs\UnityClient\MyEntity` and `Resources\Prefabs\UnityGameLogic\MyEntity`.
```

Comment:
we need to find a convention for slashes. Ive been using forward slashes in my docs, while here backward slashes have been used

Reply:
nope, will need to check with @ElleEss or we could check the spatialos docs to check what has been used there